Navigation/Signoffs/Livestream Camera Subsystem.md

```diff
@@ -0,0 +1,114 @@
+# Livestream Camera Subsystem
+# Function of the subsystem
+
+The Camera Subsystem of the Autonomous Crawlspace Inspection Robot is to capture real-time video footage of the crawlspace and transmit it to a central control unit for immediate viewing and assessment.
+
+
+# Constraints:
+
+## Constraint 1:
+The camera must be connected to a network with a stable and unrestricted bandwidth of at least 78.64 Mbps to ensure seamless video streaming without buffering or reduced video quality.
+
+### information about Constraint 1:
+Bandwidth Calculation: 
+Bandwidth = 1280 × 1024 × 30 × 10 × (1/20) = 78,643,200 bps = 78.64 Mbps. 
+The camera provides a video stream with a resolution of 1200 TVL (1280x1024) at 30 fps, using H.264 compression (compression ratio of 20:1 for H.264).
+The required bandwidth is calculated to be 78.64 Mbps. Sufficient bandwidth is essential for seamless video streaming. When the available bandwidth is lower than the required minimum, it can lead to buffering or reduced video quality. The network conditions are unstable or restricted, consistently meeting the bandwidth requirement becomes difficult and impacts the viewing.
+
+
+## Constraint 2:
+The Camera must achieve a latency below 100 milliseconds to minimize a lag of 0.05 meters in the displayed video, ensuring prompt response and efficient navigation in the crawlspace.
+
+### information about Constraint 2:
```

Comment:
We can't make assumptions in design. If it doesn't state the latency then we don't know it. What does it mean for a camera to have a latency?